php/laravel/README.md

```diff
@@ -0,0 +1,23 @@
+# Build and run
+
+API endpoints will get started on [http://127.0.0.1/8000/api]
+
+```
+php artisan serve
+```
+
+Additional details for getting started with Laravel and YugabyteDB can be found [here](Laravel.md).
+
+# Customizing
+
+There are a number of options that can be customized in the `.env` file located here:
+[`.env`](.env)
```

Comment:
Link is broken. There is no `.env` but  only `.env.example`. May be you mean to suggest users to rename that  file before editing?

Reply:
Made the required changes to fix this. .env file was part of the .gitignore so it file didn't get committed.